com.gitlab.kendellfab.restscope.json

```
@@ -0,0 +1,55 @@
+{
+    "app-id": "com.gitlab.kendellfab.restscope",
+    "runtime": "org.gnome.Platform",
+    "runtime-version": "3.34",
+    "sdk": "org.gnome.Sdk",
+    "sdk-extensions": [
+        "org.freedesktop.Sdk.Extension.dmd"
+    ],
+    "command": "restscope",
+    "finish-args": [
+        "--socket=x11",
```

```suggestion
        "--socket=fallback-x11",
```
If your application runs fine under Wayland, this will disable the X socket when not needed.